### PR TITLE
DEV-4643: bizz-card Readme

### PR DESCRIPTION
**Pull request by MSilva95**
Fixing the readme struct and adding more instructions to make the project easy to achieve

### CODE REVIEW

File: subjects/mobile-dev/bizz-card/README.md [hunk 1]
```diff
@@ -1,81 +1,88 @@
...
+- Email
+- Photo/Image
 
-You should display static information about yourself, i.e. name, surname, age, phone number, email, and image.
-Try to make it as it is shown in the example below:
+Here is an example of what we are asking:
 
 <!-- <center> -->
 <center>
 <img src="./resources/bizzCard.01.png?raw=true" style = "width: 210px !important; height: 420px !important;"/>
```
Comment (nprimo) [inline subjects/mobile-dev/bizz-card/README.md:61]: In the https://preview.01-edu.org/preview the image is not rendered - at least not in my current browser (Brave - Chromium to 113.0.5672.92.)

**Review by nprimo — CHANGES_REQUESTED** [review]
The link of a picture might need to be replaced/fixed - I left a comment in line. 
Besides that, everything else looks good to me :rocket:

File: subjects/mobile-dev/bizz-card/README.md (hunk 1 above)
Comment (MSilva95) [reply subjects/mobile-dev/bizz-card/README.md:61]: on my side its fine... not sure what is happening...
image

**Review by nprimo — APPROVED** [review]
LGTM :rocket:

**Pull request merged**